rest-api/vip-endpoints.php

```diff
@@ -123,6 +131,94 @@ public function list_sites() {
 
 		return new WP_REST_Response( $sites );
 	}
+
+	/**
+	 * Build list of active plugins on site ()
+	 */
+	public function list_plugins() {
+		$all_plugins = array();
+		$standard_plugins = array();
+		$shared_plugins = array();
+		$mu_plugins = array();
+		$client_mu_plugins = array();
+
+		if ( ! function_exists( 'get_plugins' ) ) {
+			require_once ABSPATH . 'wp-admin/includes/plugin.php';
+		}
+
+		// array of all standard plugins
+		$standard_plugins = get_plugins();
+		$tmp_plugins = array();
+		foreach ( $standard_plugins as $key => $plugin ) {
+			if ( is_plugin_active( $key ) ) {
+				$tmp_plugins[ $key ] = array(
```

Comment:
Added support for code activated standard plugins and also added inactive plugins to the output as well. All plugins have an `active` boolean too. 

It doesn't look like we are allowing theme activated plugins on Go already:

https://github.com/Automattic/vip-go-mu-plugins/blob/900fb80d9dbe4934caf014b712ca022fbbb3395e/vip-helpers/vip-utils.php#L993-L998